nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExecuteStreamCommand.java

```diff
@@ -471,6 +471,7 @@ public void onTrigger(ProcessContext context, final ProcessSession session) thro
         try {
             process = builder.start();
         } catch (IOException e) {
```

Comment:
`delete()` can throw several exceptions here, I think they should be handled.
maybe     https://github.com/apache/nifi/blob/f9ae3bb9c970cd8d6d1d9e10f07cab9bdb66baa9/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-nar-utils/src/main/java/org/apache/nifi/util/FileUtils.java#L77 ,  and the component logger passed so failures can be logged.

Reply:
Thanks for your review, I have added the exception handling

I didn't use the FileUtils you mentioned
As far as I know, components need to use ComponentLog to record logs to distinguish the components in the same instance.
But FileUtils does not support it. so I handle the exception directly.